.docker/nginx/browser.conf

```diff
@@ -1,3 +1,10 @@
+resolver 127.0.0.11;
```

Comment:
is this a standard for `docker` or `docker-compose`?

Reply:
Docker.